src/adu-shell/CMakeLists.txt

```diff
@@ -1,7 +1,5 @@
 cmake_minimum_required (VERSION 3.5)
 
-project (azure_iot_aduc_shell)
-
 set (target_name adu-shell)
```

Comment:
Nit: (can be addressed in separate PR) The other targets use underscore instead of hyphen